deploy.sh

```diff
@@ -10,18 +10,50 @@ KEY="add-keyname"
 # Posit Workbench Version
 PWB_VER=2022.07.2-576.pro12
 PWB_VER=2022.12.0-353.pro20
-PWB_VER=2023.03.0-386.pro1
+PWB_VER=2023.09.1-494.pro2
 #PWB_VER=2023.05.0-daily-312.pro2
 # SLURM Version - use only "-" to resemble git tag version
 SLURM_VER=22-05-5-1
 
 CERT="/Users/michael/projects/aws/certs/michael.pem"
 
+# create random rstudio password and store it in a secret 
+
+## rstudiopw will be a 12 character random string with mixed upper and lowercase characters + numbers
+rstudiopw=`python -c "import random,string; x=''.join(random.choices(string.ascii_uppercase + string.ascii_lowercase + string.digits, k=12));print(x)"`
```

Comment:
I defer to @aja08379 as to whether this is sufficient.

Reply:
@michaelmayer2 @jspiewak if this is coupled with the networking changes we mentioned (removing 0.0.0.0/0) then this looks fine to me. I would be surprised at this being brute-forced via the UI.